### ISSUE TITLE
Cache user activity failed calls

### ISSUE THREAD

**Issue by noumantahir** [ecency/ecency-mobile#2510]
Track failed usr-activity calls and retry them later, no need show usr if it fails, points can be awarded passively.

1. Check for useQuery implantation if we can use it for making failed calls upon app relaunch
2. If not able to use useQuery, go for manual option like cache or redux to call passive activities calls

Update: this thread suggest we might not be able to get what we want from useQuery out of the box and might still have to use some state management
https://github.com/TanStack/query/discussions/3580#discussioncomment-2708403

**Comment by noumantahir** [comment]
@feruzm quick question, if user posts a content to community and selects reblog options during publish, should user be awarded points for reblog exclusively or not. Right now, user gets awarded exclusively.


<img width="776" alt="Screenshot 2022-10-13 at 2 59 45 PM" src="https://user-images.githubusercontent.com/6298342/195567391-a947993f-dde0-4e8c-9d07-aa07b25dba05.png">
<img width="582" alt="Screenshot 2022-10-13 at 3 00 06 PM" src="https://user-images.githubusercontent.com/6298342/195567403-86771098-3ca7-4057-8e1f-f1d2d843e41a.png">

**Comment by feruzm** [comment]
@noumantahir yes should be rewarded for that reblog as well. It is likely overlooked during implementation, if we say reblog rewarded then we should stand by it, even if user reblog comments.
